apps/frontend/src/app/(onboarding)/stack/page.tsx

```diff
@@ -18,14 +22,23 @@ const StackConnect: React.FC = () => {
     }
   }, [])
 
+  useEffect(() => {
+    if (session) {
+      fetchUser(session)
+    }
+  }, [session, fetchUser])
+
+  if (error) return <div className="text-red-500">{error}</div>
+  if (!user) return <div>User not found</div>
```

Comment:
Add some color here, primary-foreground or something since on in dark mode nothing will be visible.